### PR TITLE
[wasm] Misc build+test fixes

### PR DESCRIPTION
**Pull request by radical**
- [wasm] Use `-s EXPORT_ES6=1` only for link time
- [wasm] Fix loading `dotnet.native.js.symbols`
- WBT: Re-enable previously disabled tests
- Add make target `build-packages` to build the nugets
- Browser.targets - normalize tmp webcil paths

### CODE REVIEW

File: src/mono/nuget/Microsoft.NET.Sdk.WebAssembly.Pack/build/Microsoft.NET.Sdk.WebAssembly.Browser.targets [hunk 1]
```diff
@@ -394,10 +394,11 @@ Copyright (c) .NET Foundation. All rights reserved.
     </ComputeWasmPublishAssets>
 
     <PropertyGroup>
-      <_WasmPublishWebCilPath>$(IntermediateOutputPath)webcil\publish</_WasmPublishWebCilPath>
+      <_WasmPublishWebCilPath>$([MSBuild]::NormalizeDirectory($(IntermediateOutputPath), 'webcil', 'publish'))</_WasmPublishWebCilPath>
+      <_WasmPublishTmpWebCilPath>$([MSBuild]::NormalizeDirectory($(IntermediateOutputPath), 'tmp-webcil', 'publish'))</_WasmPublishTmpWebCilPath>
     </PropertyGroup>
 
-    <ConvertDllsToWebCil Candidates="@(_NewWasmPublishStaticWebAssets)" IntermediateOutputPath="$(IntermediateOutputPath)tmp-webcil\publish" OutputPath="$(_WasmPublishWebCilPath)" IsEnabled="$(_WasmEnableWebcil)">
+    <ConvertDllsToWebCil Candidates="@(_NewWasmPublishStaticWebAssets)" IntermediateOutputPath="$(_WasmPublishTmpWebCilPath)" OutputPath="$(_WasmPublishWebCilPath)" IsEnabled="$(_WasmEnableWebcil)">
```
Comment (maraf) [inline src/mono/nuget/Microsoft.NET.Sdk.WebAssembly.Pack/build/Microsoft.NET.Sdk.WebAssembly.Browser.targets:401]: The task is called twice (build & publish). Should the other place be also updated?

File: src/mono/wasm/runtime/loader/polyfills.ts [hunk 2]
```diff
@@ -93,7 +93,8 @@ export async function fetch_like(url: string, init?: RequestInit): Promise<Respo
                 },
                 json: () => {
                     return JSON.parse(read(url, "utf8"));
-                }
+                },
+                text: () => read(url, "utf8")
```
Comment (maraf) [inline src/mono/wasm/runtime/loader/polyfills.ts:97]: Should we polyfill the `text` function also for node & error case?

File: src/mono/wasm/runtime/loader/assets.ts [hunk 3]
```diff
@@ -168,9 +164,12 @@ export async function mono_download_assets(): Promise<void> {
...
                         }
                     } else {
-                        if (skipBufferByAssetTypes[asset.behavior]) {
-                            ++loaderHelpers.actual_downloaded_assets_count;
+                        if (asset.behavior === "symbols") {
+                            await runtimeHelpers.instantiate_symbols_asset(asset);
+                            cleanupAsset(asset);
                         }
+
+                        ++loaderHelpers.actual_downloaded_assets_count;
```
Comment (pavelsavara) [inline src/mono/wasm/runtime/loader/assets.ts:172]: I would prefer to keep the condition
`if (skipBufferByAssetTypes[asset.behavior])`

**Review by pavelsavara — APPROVED** [review]

File: src/mono/wasm/runtime/loader/assets.ts (hunk 3 above)
Comment (pavelsavara) [reply src/mono/wasm/runtime/loader/assets.ts:172]: the idea is that those *ByAssetTypes collections maintain what should happen, not the code/conditions. And in some cases that's not perfect. Feel free to improve it.

File: src/mono/wasm/runtime/loader/polyfills.ts (hunk 2 above)
Comment (radical) [reply src/mono/wasm/runtime/loader/polyfills.ts:98]: done

File: src/mono/nuget/Microsoft.NET.Sdk.WebAssembly.Pack/build/Microsoft.NET.Sdk.WebAssembly.Browser.targets (hunk 1 above)
Comment (radical) [reply src/mono/nuget/Microsoft.NET.Sdk.WebAssembly.Pack/build/Microsoft.NET.Sdk.WebAssembly.Browser.targets:402]: good catch! done.

**Pull request merged**